### PR TITLE
Added checklist and recommendations to issue template

### PR DESCRIPTION
**Pull request by asmorkalov**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under OpenCV (BSD) License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [x] There is reference to original bug report and related work
- [x] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by asmorkalov** [comment]
Ticket example: https://github.com/asmorkalov/opencv/issues/6

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -36,3 +21,40 @@ This is a template helping you to create an issue which can be processed as quic
     ```
  or attach as .txt or .zip file
 -->
+
+##### Issue submission checklist
+
+ - [ ] I report the issue, it's not a question
+   <!--
+   OpenCV team works with answers.opencv.org, Stackoverflow and other communities
+   to disucss problems. Tickets with question without real probelem statement will
```
Comment (alalek) [inline .github/ISSUE_TEMPLATE.md]:
```.diff
-Stackoverflow
+StackOverflow
```

Ref: https://twitter.com/StackOverflow

> disucss
> probelem

typos

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -36,3 +21,40 @@ This is a template helping you to create an issue which can be processed as quic
...
+   be closed.
+   -->
+ - [ ] I checked the problem with documentation, FAQ, open issues,
+       answers.opencv.org, Stackoverflow, etc and have not found solution
+   <!--
+   Places to check:
+   * FAQ page: https://github.com/opencv/opencv/wiki/FAQ,
+   * OpenCV forum: https://answers.opencv.org
+   * Stackoverflow branch: https://stackoverflow.com/questions/tagged/opencv
+   -->
```
Comment (alalek) [inline .github/ISSUE_TEMPLATE.md]: Add this link:
`https://github.com/opencv/opencv/issues?q=is%3Aissue`

(parameter is required to show closed issues too)

**Comment by asmorkalov** [comment]
@alalek Fixed commends, added links to OpenCV issue tracker and documentation.

**Comment by asmorkalov** [comment]
@alalek @vpisarev any comments on the new version?

**Pull request merged**